CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [1.17.3]
+### Changed
+- `sockeye.evaluate` can now handle multiple hypothesis.
```

Comment:
"hypotheses".
Comment could also include how to supply them :)

Reply:
done